packages/mikado_reborn/src/utils/themes.js

```diff
@@ -0,0 +1,6 @@
+const themes = {
```

Comment:
We should export them from the related component. This export can be misleading because this only refers to the interactive icon.

Reply:
Nice catch, did not see each component had its own theme variables !